ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_search.py

```diff
@@ -295,3 +295,38 @@ def test_paddle_topk(
         sorted=sorted,
         test_values=False,
     )
+
+
+# where
+@handle_frontend_test(
+    fn_tree="paddle.where",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        min_num_dims=1,
+    ),
+    test_with_out=st.just(False),
+)
+def test_paddle_where(
+    *,
+    dtype_and_x,
+    on_device,
+    fn_tree,
+    frontend,
+    backend_fw,
+    test_flags,
+    condition,
+    x,
+    y,
```

Comment:
The test still fails because these three values aren't defined in your `@handle_frontend_test` above. Check how other frontends test their `where` functions, and also make sure you're able to run the tests locally